__tests__/app.test.js

```diff
@@ -131,5 +131,76 @@ describe("News Api Tests",()=>{
         })
+    }
+    describe("GET/API/ARTICLES", ()=>{
+        describe("Happy Path", ()=>{
+            test("200: Returns an okay Status Code", ()=>{
+                return request(app)
+                .get("/api/articles")
+                .expect(200);
+            })
+            test("200: Returns all articles with correct key value pairs (except the comment count)", ()=>{
+                const desiredArticle = {
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: '2020-07-09T20:11:00.000Z',
+                    votes: 100,
+                    article_img_url:
+                    "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                }
+                
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body})=>{
+                    const { articles } = body
+                    const lastArticle = articles[12]
+
+                    expect(articles).toHaveLength(13)
+                    expect(lastArticle).toMatchObject(desiredArticle)
+                    
+                    articles.forEach((article)=>{
+                        expect(article).toMatchObject({
+                            article_id: expect.any(Number),
+                            title: expect.any(String),
+                            topic: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            article_img_url: expect.any(String)
+                        })
+                    })
+                    
+                    
+                })
+            })
+            test("200: Returns the count of the comments", ()=>{
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body})=>{
+                    const {articles} = body
+                    const firstArticle = articles[0]
+                    const lastArticle = articles[12]
+                    expect(firstArticle.comment_count).toBe(0)
+                    expect(lastArticle.comment_count).toBe(11)
+                })
+            })
+        })
+        
+        describe("Sad Path", ()=>{
+            test("404: Returns with an error message when the path is not found", ()=>{
+                return request(app)
+                .get("/api/not-a-route")
+                .expect(404)
+                .then(({body})=> {
+                    expect(body.msg).toBe("Does not exist")
+                })
+            });
+        })
```

Comment:
This test isn't needed as you already have a non existent route test added previously